wgengine/netstack/netstack.go

```diff
@@ -704,6 +708,43 @@ func (ns *Impl) injectInbound(p *packet.Parsed, t *tstun.Wrapper) filter.Respons
 	return filter.DropSilently
 }
 
+// shouldHandlePing returns whether or not netstack should handle an incoming
+// ICMP echo request packet, and the IP address that should be used to make the
```

Comment:
the IP address that should be pinged, no? This makes it sound like the source address?